src/main/app/case/case.ts

```diff
@@ -633,6 +633,10 @@ export interface Case {
   ra_languageReqAndSpecialArrangements?: string;
   ra_existingFlags?: RAFlags;
   finalServedApplicationDetailsList?: ServedApplicationDetails[];
+  wishToRespond?: YesOrNo;
+  your_response_to_aoh?: string;
+  aoh_wishToRespond?: YesOrNo;
+  aoh_responseToAllegations?: string;
```

Comment:
can we bring symmetry to the variables please? either with _ or without please?

Reply:
The standards we followed for C100 is modulename_filedName

Reply:
The standards we followed for C100 is modulename_filedName. So following the same here.